src/azure-cli/azure/cli/command_modules/monitor/commands.py

```diff
@@ -81,6 +81,16 @@ def load_command_table(self, _):
         client_factory=cf_diagnostics,
         exception_handler=monitor_exception_handler)
 
+    log_alerts_sdk = CliCommandType(
+        operations_tmpl='azure.mgmt.monitor.operations#ScheduledQueryRulesOperations.{}',
+        client_factory=cf_log_alerts,
+        exception_handler=monitor_exception_handler)
```

Comment:
is there any `min_api` requirement?

Reply:
@shpimpal , is there a min-api requirement?